### DISCUSSION TITLE
About my account being label as spam.

### DISCUSSION THREAD

**Discussion by ysfcndgr** [bluesky-social/atproto#1951] [General Discussion]
I opened an account that I created in BlueSky. and I have to admit, I wrote a script that follow other users by writing a javascript plugin. A day later, I was informed by bluesky that if I continued this, my account would be marked as spam. I didn't know this and as soon as I saw this, I stopped the script and did all my subsequent operations manually. And the same email came again and my account was label as spam. Unfortunately, there is no response to my emails. and I request that this be corrected.

Thank YOU

**Comment by Zero3K** [comment]
Why are you following a bunch of users?
